src/lib.rs

```diff
@@ -139,6 +139,45 @@ pub(crate) const CRINGE_ESLINT_DISABLE: &str = "/* eslint-disable */
 // #[cfg(doctest)]
 // doc_comment::doctest!("../README.md");
 
+/// The configuration for the generator
+#[derive(Default)]
+pub struct ExportConfiguration {
+    /// The name of the plugin to invoke.
+    ///
+    /// If there is no plugin name (i.e. this is an app), this should be `None`.
+    pub(crate) plugin_name: Option<Cow<'static, str>>,
+    /// The specta export configuration
+    pub(crate) inner: SpectaExportConfiguration,
+}
+
+impl ExportConfiguration {
+    /// Creates a new [`ExportConfiguration`] with the given plugin name.
+    pub fn new() -> Self {
+        Self::default()
+    }
+
+    /// Sets the plugin name for this [`ExportConfiguration`].
+    pub fn plugin_name(mut self, plugin_name: impl Into<Cow<'static, str>>) -> Self {
+        self.plugin_name = Some(plugin_name.into());
+        self
+    }
+
+    /// Sets the [`SpectaExportConfiguration`] for this [`ExportConfiguration`].
```

Comment:
I think this method should be removed as the idea is you convert a `specta::ts::ExportConfiguration` into an `tauri_specta::ExportConfiguration`.